pkg/virt-controller/services/template_test.go

```diff
@@ -425,7 +425,7 @@ var _ = Describe("Template", func() {
 			)
 		})
 		Context("with SELinux types", func() {
-			It("should run under the SELinux type virt_launcher.process if none specified", func() {
+			DescribeTable("should run under the SELinux type virt_launcher.process", func(hugePages, virtiofs bool) {
```

Comment:
Shouldn't you add a passt case here?

nit: I would also refactor the bool approach and pass a closure to mutate the VMI (much nicer I think)

Reply:
Passt doesn't need virt_launcher.process, it'll run as container_t. Maybe we could have a test with a VMI that has everything *but* virtofs and hugepages and ensures we don't set an SELinux type on it, but that sounds a bit silly...
I used bools because this test doesn't use libvmi and I don't think it worth refactoring it just for that, but I can if you feel strongly about it.
Thanks for the review!

Reply:
No need, I just misunderstood your comment https://github.com/kubevirt/kubevirt/pull/8402#discussion_r986132932.
Thanks!